Automattic/woocommerce-payments
# Add dispute resolution task

Fixes #1658

#### Changes proposed in this Pull Request

This PR adds a dispute resolution task for each dispute that the users have.

![screenshot-fermarichal jurassic tube-2021 06 11-10_56_43](https://user-images.githubusercontent.com/1314156/121697866-dbbcde00-caa3-11eb-8123-e8576ae61b10.png)










#### Testing instructions

- Create a new dispute. You can do it by buying one product with one of the [Stripe test disputes cards](https://stripe.com/docs/testing#disputes) (E.g.: `4000000000001976` use any date in the future and fill out `CVC` with any number).
- Verify the information displayed on the card is correct.
-  Click on the card. You should be redirected to the dispute's overview screen.
- Create another dispute and verify that the new dispute is also visible as a new dispute resolution task.
- Click on the new card and verify that is working as expected.





-------------------


-------------------

### UPDATE

After merging [PR 2053](https://github.com/Automattic/woocommerce-payments/pull/2053), WCPay uses [@woocommerce/experimental 1.3.0](https://github.com/Automattic/woocommerce-payments/blob/develop/package.json#L101). A few modifications were added to the task list in that version that affects this PR and the [reconnect WooCommerce Payments](https://github.com/Automattic/woocommerce-payments/pull/1912) task.
The [issue 7199](https://github.com/woocommerce/woocommerce-admin/issues/7199) was created to fix it.

### UPDATE 2

Please review this PR after merging [PR 7263](https://github.com/woocommerce/woocommerce-admin/pull/7263), [PR 7300](https://github.com/woocommerce/woocommerce-admin/pull/7300), and having a new version of  `@woocommerce/experimental` released.
